integrations/transportation/tripit.js

```diff
@@ -148,6 +166,69 @@ async function fetchRail(modifiedSince, isPast = true, logger) {
   return { activities: flatten(activities), modifiedSince: data.timestamp };
 }
 
+async function fetchLodging(modifiedSince, isPast = true, logger) {
+  const url = modifiedSince
+    ? `/v1/list/object/type/lodging/past/${isPast}/format/json/modified_since/${modifiedSince}`
+    : `/v1/list/object/type/lodging/past/${isPast}/format/json`;
+  const res = await manager.fetch(url);
+  if (!res.ok) {
+    const text = await res.text();
+    throw new HTTPError(text, res.status);
+  }
+  const data = await res.json();
+  const pageNum = parseInt(data['page_num'], 10);
+  const pageMax = parseInt(data['max_page'], 10);
+
+  if (pageMax > pageNum) {
+    throw Error('Not implemented pagination is required');
+  }
+
+  let objects = data.LodgingObject;
+  if (!objects) {
+    return { activities: [], modifiedSince: data.timestamp };
+  }
+  if (!Array.isArray(objects)) {
+    objects = [objects];
+  }
+
+  const activities = objects.map((s) => {
+    try {
+      const [startDate, endDate] = [
+        parseDatetime(s.StartDateTime),
+        parseDatetime(s.EndDateTime),
+      ];
+      const durationHours = (endDate.getTime() - startDate.getTime()) / 1000.0 / 3600.0;
```

Comment:
Why the `1000.0` and `3600.0`? Doesn't make much sense and I can't see any linting rules that makes it a requirement :)

Reply:
it's for readability: / 1000 turns ms into s, and / 3600 turns s into hours

Reply:
Sorry, my comment wasn't clear - I'm specifically talking about the `.0` part on numbers :)

Reply:
ah! It's remnants from python where 0.0 is a float and 0 is an integer.
Adding the .0 forces to avoid integer division which can cause unexepected rounding.

Reply:
Ah okay, makes sense. Not blocking anything, so let's leave it like this for now.